realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy

```diff
@@ -278,4 +288,23 @@ class RealmTransformer extends Transform {
         return merged;
     }
 
+    // Use reflection to get the AndroidBuild from Android plugin for getting the path to android.jar
+    // There is no official way to get the path to android.jar for transform.
+    // See https://android.googlesource.com/platform/tools/base/+/gradle_2.0.0/build-system/gradle-core/src/main/groovy/com/android/build/gradle/internal/transforms/InstantRunTransform.java#337
+    private void addBootClassesToClassPool(ClassPool classPool) {
+        try {
+            def androidBuilderField = BasePlugin.getDeclaredField("androidBuilder");
```

Comment:
Do we know if this works in the Experimental plugin? Nothing we can do about it if it doesn't right now, but at least having a TODO here just in case then?